python/dgl/transforms/functional.py

```diff
@@ -3300,4 +3302,69 @@ def norm_by_dst(g, etype=None):
 
     return norm
 
+def rwpe(g, k):
+    r"""Return random walk positional encodings.
+
+    For internal use.
```

Comment:
This API will be visible to users if you want to add it. It's just a functional interface while `RWPE` is a module interface.

Reply:
done